main/guides/getting-started/Explainer - How to Start a Local Chain.md

```diff
@@ -0,0 +1,57 @@
+# Starting a Local Chain
```

Comment:
We cover this on `/getting-started` already. Let's please not duplicate it.

Reply:
This is what @sufyaankhan wanted documented...